lib/widgets/registerorgform.dart

```diff
@@ -33,7 +33,7 @@ class _RegisterOrgFormState extends State<RegisterOrgForm> {
       "Signing up Admin",
       null,
       "Failed to create ${emailCtrl.text} as admin"
-    ) as bool? ?? false;
+    ) != null;
```

Comment:
we really should setup the loadingdialog to take true/false instead of null/not null. But idk how much work that would be. Non blocking change.

Reply:
I’m okay with it. I guess my brain isn’t used to all the dart null safety checks being around the program. So I guess nulls being thrown around isn’t a big deal like in over languages.